package.json

```diff
@@ -28,6 +28,12 @@
   },
   "babel": {
     "plugins": [
+      [
+        "transform-runtime",
+        {
+          "polyfill": false
```

Comment:
`babel-plugin-transform-runtime` is used instead of `babel-polyfill` due to
https://github.com/OctoLinker/browser-extension/pull/258#issuecomment-275813737

Note one of them is required. `babel-preset-env` alone isn't enough:
https://github.com/babel/babel/issues/5085#issuecomment-271640887